src/import/instruments.ts

```diff
@@ -361,7 +361,11 @@ async function readFormTemplate(tree: TraceDirectoryTree): Promise<FormTemplateD
   const archive = readInstrumentsKeyedArchive(await readAsArrayBuffer(formTemplate))
 
   const version = archive['com.apple.xray.owner.template.version']
-  const selectedRunNumber = archive['com.apple.xray.owner.template'].get('_selectedRunNumber')
+  let selectedRunNumber = 1
+  if ('com.apple.xray.owner.template' in Object.keys(archive)) {
+    selectedRunNumber = archive['com.apple.xray.owner.template'].get('_selectedRunNumber')
+    console.log('get selectedRunNumber', selectedRunNumber)
```

Comment:
I'm guessing this was useful for debugging, but can you please remove this before we merge?

Reply:
- `.tracetemplate` file create by Instruments `File` -> `Save As Template`
- `console.log('get selectedRunNumber', selectedRunNumber)` is debug code and have removed
- add a testing with small `.trace` file that without selected run number

Thanks for your suggestion

Reply:
- `.tracetemplate` file create by Instruments `File` -> `Save As Template`
- `console.log('get selectedRunNumber', selectedRunNumber)` is debug code and have removed
- add a testing with small `.trace` file that without selected run number

Thanks for your suggestion